<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1056</width>
    <height>764</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Traveling Salesman Problem</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/img/kvkghforv9kxb9zntjdkqi7xdlw.jpeg</normaloff>:/img/kvkghforv9kxb9zntjdkqi7xdlw.jpeg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Put">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>17</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
background-color:rgb(114, 159, 207);
}</string>
    </property>
    <property name="text">
     <string>Путь: </string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>501</width>
      <height>471</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGraphicsView {
border: 2px solid gray;
background-color: rgba(255, 255, 255, 50);

}

</string>
    </property>
   </widget>
   <widget class="QLabel" name="DPut">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>540</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
      <pointsize>17</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
background-color:rgb(114, 159, 207);
}</string>
    </property>
    <property name="text">
     <string>Длина пути: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>540</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Monaco</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
background-color:rgb(114, 159, 207);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Marshrut">
    <property name="geometry">
     <rect>
      <x>36</x>
      <y>500</y>
      <width>475</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Monaco</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
background-color:rgb(114, 159, 207);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="res">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Monaco</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border-color: rgb(128, 72, 131);
background-color: rgb(173, 127, 168);
border: 2px solid gray;
}
QPushButton:pressed {
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
stop: 0 #dadbde, stop: 1 #87CEEB);
}
QPushButton:hover{
border-color: rgb(47, 48, 46);
background-color: rgb(114, 159, 207);
border-radius: 10px;
border: 1px solid #3873d9;
}</string>
    </property>
    <property name="text">
     <string>Решить задачу</string>
    </property>
   </widget>
   <widget class="QPushButton" name="O4ISTKA">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Monaco</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border-color: rgb(128, 72, 131);
background-color: rgb(177, 78, 78);
border: 2px solid gray;
}
QPushButton:pressed {
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
stop: 0 #dadbde, stop: 1 #87CEEB);
}

QPushButton:hover{
border-color: rgb(47, 48, 46);
background-color: rgb(245, 121, 0);
border-radius: 10px;
border: 1px solid #3873d9;
}</string>
    </property>
    <property name="text">
     <string>Очистить всё</string>
    </property>
   </widget>
   <widget class="QPushButton" name="VblvodP">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Monaco</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border-color: rgb(128, 72, 131);
background-color: rgb(173, 127, 168);
border: 2px solid gray;
}
QPushButton:pressed {
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
stop: 0 #dadbde, stop: 1 #87CEEB);
}
QPushButton:hover{
border-color: rgb(47, 48, 46);
background-color: rgb(114, 159, 207);
border-radius: 10px;
border: 1px solid #3873d9;
}
</string>
    </property>
    <property name="text">
     <string>Построить</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="planet">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Corbel</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
qproperty-alignment: 'AlignVCenter | AlignMid';
border: 1px solid gray;
  background-color: rgb(233, 204, 238);

}
</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
qproperty-alignment: 'AlignVCenter | AlignMid';
border: 1px solid gray;
  background-color: rgb(233, 204, 238);
}</string>
    </property>
    <property name="text">
     <string>Количество планет: </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>50</y>
      <width>171</width>
      <height>151</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
border-color: rgb(128, 72, 131);
background-color: rgba(173, 127, 168,150);
border: 2px solid gray;
}


QPushButton:pressed {
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
stop: 0 #dadbde, stop: 1 #87CEEB);
}</string>
    </property>
    <property name="text">
     <string>И</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>200</y>
      <width>51</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
border-color: rgb(128, 72, 131);
background-color: rgb(173, 127, 168);
border: 2px solid gray;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>220</y>
      <width>131</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
border-color: rgb(128, 72, 131);
background-color: rgb(173, 127, 168);
border: 2px solid gray;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_put">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>150</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_iz">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>100</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_v">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>100</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>50</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
qproperty-alignment: 'AlignVCenter | AlignMid';
}</string>
    </property>
    <property name="text">
     <string>Добавить путь</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>70</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
qproperty-alignment: 'AlignVCenter | AlignMid';
}</string>
    </property>
    <property name="text">
     <string>между планетами</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>150</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>  Путь =</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>240</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
border-color: rgb(128, 72, 131);
background-color: rgb(173, 127, 168);
border: 2px solid gray;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>350</y>
      <width>51</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
border-color: rgb(128, 72, 131);
background-color: rgb(173, 127, 168);
border: 2px solid gray;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>350</y>
      <width>51</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
border-color: rgb(128, 72, 131);
background-color: rgb(173, 127, 168);
border: 2px solid gray;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>350</y>
      <width>51</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
border-color: rgb(128, 72, 131);
background-color: rgb(173, 127, 168);
border: 2px solid gray;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="FON">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1021</width>
      <height>601</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Galaxy123.png</pixmap>
    </property>
   </widget>
   <zorder>FON</zorder>
   <zorder>planet</zorder>
   <zorder>label_8</zorder>
   <zorder>label_4</zorder>
   <zorder>label</zorder>
   <zorder>Marshrut</zorder>
   <zorder>Put</zorder>
   <zorder>graphicsView</zorder>
   <zorder>DPut</zorder>
   <zorder>res</zorder>
   <zorder>O4ISTKA</zorder>
   <zorder>VblvodP</zorder>
   <zorder>label_2</zorder>
   <zorder>pushButton</zorder>
   <zorder>label_3</zorder>
   <zorder>lineEdit_put</zorder>
   <zorder>lineEdit_iz</zorder>
   <zorder>lineEdit_v</zorder>
   <zorder>label_5</zorder>
   <zorder>label_6</zorder>
   <zorder>label_7</zorder>
   <zorder>label_9</zorder>
   <zorder>label_10</zorder>
   <zorder>label_11</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
